docker/wait-for-db.sh

```diff
@@ -12,21 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-# Usage: $ ./wait-for-db.sh <host> <port>
+# Usage: $ ./wait-for-db.sh <host> <port> [timeout]
 
 set -eu
 
 host="${1}"
 port="${2}"
+time="${3}" || 60s
```

Comment:
_Minor_: Let's default the `time` param as:

```
time="${3:-60s}"
```

Reply:
Done.